api/swagger.json

```diff
@@ -73,7 +73,72 @@
         }
       }
     },
-    "/mesa": {
+    "/voting-tables/{id}/reportarFaltaFiscal": {
```

Comment:
Shouldn't all the endpoints be in English (following the nature of this PR)?

Reply:
@brunocalmels Solved